### DISCUSSION TITLE
Help! How to relate images to components

### DISCUSSION THREAD

**Discussion by ManjorStreep** [strapi/strapi#8883] [Legacy Topics]
How did you relate the uploaded images to this component?

![Screenshot_196](https://user-images.githubusercontent.com/61101240/102003847-65504680-3ce1-11eb-8c44-0e0ff97a4b92.png)

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
